databook/background/Mouse-visual-system.md

```diff
@@ -45,6 +45,26 @@ visual areas each contain a distinct map of {term}`Retinotopy`, i.e. each visual
 area has its own map of visual space. The borders between higher visual areas
 can be identified based on reversals in the retinotopic map at area boundaries.
 
+The retinotopic map is often studied using intrinsic signal imaging (ISI), which
+utilizes the difference in the absorption spectra between deoxygenated and
+oxygenated hemoglobin. The cortical surface is exposed to red light and the
+subsequent reflection is measured using a CCD sensor. This allows researchers 
+to identify where oxygenated blood is being diverted, allowing for identification
+of active regions, albeit with relatively low resolution (~200 $\mu$m, ~1-2 s).
+
+To create a retinotopic map, mice are exposed to drifting stimuli while ISI
```

Comment:
Measuring this activity while mice view a drfiting stimulus, we can create a map of altitude and azimuth across the cortical surface.

Reply:
And then we need a better description of how we get from azimuth/altitude to visual cortical boundaries. I don't have the right language off the top of my head, but Michael can point you to our papers or Marina's papers.